src/logic/safe/utils/modules.ts

```diff
@@ -65,26 +60,28 @@ export const getModules = async (safeInfo: SafeInfo | void): Promise<Array<Modul
     return
   }
 
+  const safeInfoModules = safeInfo.modules.map(({ value }) => value)
+
   if (semverLessThan(safeInfo.version, '1.1.1')) {
```

Comment:
Is this if else logic required? The backend should always return all modules.

Maybe check with @Uxio0 on that and also make sure that the backend always returns the correct order (as order is important for this logic here)

Reply:
If I'm not wrong, `gnosis-py` is only using `getModules` https://github.com/gnosis/gnosis-py/blob/master/gnosis/safe/safe.py#L564

Reply:
Fair, I would still not add this logic here ... then better open an issue on the gnosis py to add support for this.

Reply:
Yes, agree. This was here until the change in gnosis-py was made.

I'll open the issue and add a TODO in this block pointing to it.

Thanks!

Reply:
Unless `10` modules are returned so we don't know if there're more, and then we fallback to the paginated version https://github.com/gnosis/gnosis-py/blob/master/gnosis/safe/safe.py#L571 😉

Reply:
:scream: ahhh... I'll close the issue I just open :grimacing: